src/app/state/rest/provider/ApiProvider.tsx

```diff
@@ -19,7 +19,8 @@ export const ApiContext = React.createContext<GroupedContext>({
   teamSettings: noopContext,
   teamSettingsList: noopContext,
   postCodeRangesPresets: noopContext,
-  auth: noopContext
+  auth: noopContext,
+  meldingen: noopContext
```

Comment:
Is het niet handiger om `noopContext` zelf toe te voegen aan de context en dat consumers die dan direct gebruiken?